CMakeLists.txt

```diff
@@ -29,38 +33,54 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_AUTOUIC ON)
 
-if(CMAKE_VERSION VERSION_LESS "3.7.0")
-    set(CMAKE_INCLUDE_CURRENT_DIR ON)
-endif()
-
-set(RELEASE_NAME "Makou_Reactor")
-set(RELEASE_VERSION "1.8.2")
-set(GUI_TARGET "${RELEASE_NAME}")
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-find_package(Qt5 COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
+find_package(QT NAMES Qt5 COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
 find_package(ff7tk REQUIRED NO_MODULE COMPONENTS ff7tk ff7tkUtils)
 find_package(ZLIB REQUIRED)
 
+file(GLOB_RECURSE TS_FILES "${RELEASE_NAME}_*.ts")
 file(GLOB_RECURSE PROJECT_SOURCES "*.cpp" "*.h")
-
-# Translations
-file(GLOB_RECURSE translation_files "${RELEASE_NAME}_*.ts")
-set_source_files_properties(${translation_files} PROPERTIES OUTPUT_LOCATION "${CMAKE_INSTALL_FULL_BINDIR}")
-qt5_add_translation(QM_FILES ${translation_files})
-
-if(WIN32)
-    set (FF7TK_DEPENDS
-        ${ff7tk_DIR}/../../../bin/ff7tk.dll
-        ${ff7tk_DIR}/../../../bin/ff7tkWidgets.dll
-    )
-endif()
+set(RESOURCES "src/qt/${RELEASE_NAME}.qrc")
 
 # GUI
-add_executable(${GUI_TARGET} ${PROJECT_SOURCES} ${QM_FILES} src/qt/${RELEASE_NAME}.qrc src/qt/${RELEASE_NAME}.rc)
+add_executable(${GUI_TARGET} ${PROJECT_SOURCES} ${QM_FILES} ${RESOURCES} src/qt/${RELEASE_NAME}.rc)
 target_include_directories(${GUI_TARGET} PRIVATE "${CMAKE_SOURCE_DIR}/src")
 target_link_libraries(${GUI_TARGET}
-    Qt5::OpenGL
-    Qt5::Widgets
+    Qt${QT_VERSION_MAJOR}::OpenGL
+    Qt${QT_VERSION_MAJOR}::Widgets
     ff7tk::ff7tk
     ff7tk::ff7tkUtils
+    ZLIB::ZLIB
 )
+
+# CLI
+#add_executable(${CLI_TARGET} ${PROJECT_SOURCES} ${QM_FILES} src/qt/${RELEASE_NAME}.qrc src/qt/${RELEASE_NAME}.rc)
+#target_include_directories(${CLI_TARGET} PRIVATE "${CMAKE_SOURCE_DIR}/src")
+#target_link_libraries(${CLI_TARGET}
+#    Qt${QT_VERSION_MAJOR}::Widgets
+#    ff7tk::ff7tk
+#    ff7tk::ff7tkUtils
+#)
+#target_compile_definitions(${CLI_TARGET}
+#    PRIVATE MR_CONSOLE=1
+#)
+
+qt5_create_translation(QM_FILES ${CMAKE_SOURCE_DIR} ${TS_FILES})
```

Comment:
Becareful here without making them in their own directory you may find on clean your TS files are deleted.
see : https://github.com/sithlord48/ff7tk/tree/master/lang

Reply:
qt5_create_translations does not work on qt6 .. you should use qt_ in place of any qt5_ or qt6_ calls if you plan to allow buildng with both versions